.github/workflows/build-fcos-image.yml

```diff
@@ -0,0 +1,57 @@
+name: "Build and push fedora-coreos container image with customzied initrd"
+
+env:
+  NAME: "fedora-coreos"
+  REGISTRY: "quay.io/confidential-clusters"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'fedora-coreos/**'
+      - '.github/workflows/build-fcos-image.yml'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'fedora-coreos/**'
+      - '.github/workflows/build-fcos-image.yml'
+  workflow_dispatch:
+
+permissions: read-all
+
+# Prevent multiple workflow runs from racing to ensure that pushes are made
+# sequentialy for the main branch. Also cancel in progress workflow runs for
+# pull requests only.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  build-fcos-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Build FCOS image
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ env.NAME }}
+          tags: latest ${{ github.sha }}
+          containerfiles: |
+            fedora-coreos/Containerfile
+          context: fedora-coreos
+          oci: true
+
+      - name: Push to Container Registry
+        uses: redhat-actions/push-to-registry@v2
+        if: (github.event_name == 'push' || github.event_name == 'schedule') && github.ref == 'refs/heads/main'
```

Comment:
This should include `workflow_dispatch` instead of `schedule` here

Reply:
Update, in https://github.com/confidential-clusters/investigations/pull/34  Thank you very much!